runtime/compiler/control/JITServerCompilationThread.cpp

```diff
@@ -1404,6 +1404,11 @@ TR::CompilationInfoPerThreadRemote::deleteClientSessionData(uint64_t clientId, T
          TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer,"Message to delete Client (%llu) received. Data for client deleted", (unsigned long long)clientId);
          }
       }
+   else if (TR::Options::getVerboseOption(TR_VerboseJITServerConns))
```

Comment:
If the verbose={JITServer} option is used, then the message that was supposed to appear due to XX:+JITServerLogConnections is ignored. This may create some confusion.